src/bluesky/callbacks/zmq.py

```diff
@@ -102,23 +133,40 @@ class Proxy:
     """
     Start a 0MQ proxy on the local host.
 
+    The addresses can be specified flexibly.  It is best to use
+    a domain_socket (available on unix):
+
+     - ``'icp:///tmp/domain_socket'``
```

Comment:
Just as FYI: for Windows users of pyzmq there's an [outstanding issue](https://github.com/zeromq/pyzmq/issues/2026) where IPC protocol is not supported by default. The issue reports how to locally build the binary to enable it but it still might cause crashes until a new release of libzmq is available which fixes this. It would be worth adding in the docstring (or in the documentation) that such limit exists.